system-x/services/microsoft/src/main/java/software/tnb/microsoft/account/MicrosoftAccount.java

```diff
@@ -0,0 +1,48 @@
+package software.tnb.microsoft.account;
+
+import software.tnb.common.account.Account;
+import software.tnb.common.account.WithId;
+
+public class MicrosoftAccount implements Account, WithId {
+    private String username;
+    private String client_id;
```

Comment:
does it work with camel case? this is a bad naming convention, but if there are technical reasons, I'm fine with it

Reply:
no it doesn't.. I tried it initially and was pretty difficult to find out that this was the issue

Reply:
it would work if:
a) I rename credentials and all the corresponding code from e.g. `client_id` to `clientId`
b) I use e.g. `clientId`, with setter like `setClient_id` and getter `clientId`